schemas/hub/definitions.json

```diff
@@ -14,58 +14,44 @@
           "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
         }
       },
-      "complete": {
+      "submission": {
         "type": "object",
         "properties": {
+          "button": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The text that appears on the submit button, when the questionnaire is ready to be submitted"
+          },
+          "guidance": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The text guidance that appears, when the questionnaire is ready to be submitted"
+          },
           "title": {
             "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
             "description": "The title that will appear on the hub, when the questionnaire is complete."
           },
-          "guidance": {
-            "$ref": "https://eq.ons.gov.uk/hub/definitions.json#/guidance"
+          "warning": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "The warning that will appear on the hub, when the questionnaire is complete."
           }
         },
+        "additionalProperties": false,
         "anyOf": [
+          {
+            "required": ["button"]
+          },
```

Comment:
I don't think any of these properties are required, as they either have a default in runner or are optional (guidance).

Reply:
My thought was that if you included the submission key it couldn't be empty, it need one child

Reply:
Fair enough, I hadn't thought about it like that.

Reply:
We should use `"minProperties": 1` instead as that is intended for the behaviour we are trying to protect against.